ap_src/ap_app/utils/user_profile.py

```diff
@@ -0,0 +1,15 @@
+# Models
+from ..models import (UserProfile)
+from django.contrib.auth.models import User
+
+def get_user_id_from_username(selected_username):
+    user_matching_username = User.objects.get(username=selected_username)
```

Comment:
I should have mentioned this previously, but when using .get(), if it can't find anything I believe it's going to throw an exception so wrapping it in a try, except is worthwhile.

Reply:
Like mentioned earlier, there is a small chance that the user's username is not in the database. This will be addressed with `try` statement.

Reply:
Fixed with ccaa924